src/middlewares/logger.ts

```diff
@@ -1,11 +1,43 @@
-import { NextFunction, Request, Response } from 'express';
+import { ErrorRequestHandler, NextFunction, Request, Response } from 'express';
+import winston from 'winston';
+
+const logger: winston.Logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.simple(),
+  transports: [
+    new winston.transports.File({ filename: 'error.log', level: 'error' }),
+    new winston.transports.File({ filename: 'combined.log' }),
+  ],
+});
+
+if (process.env.NODE_ENV !== 'production') {
+  logger.add(new winston.transports.Console());
+}
 
 export const loggerMiddleware: (req: Request, resp: Response, next: NextFunction) => void = (
   req: Request,
   resp: Response,
   next: NextFunction,
 ) => {
-  // eslint-disable-next-line no-console
-  console.log('Request logged:', req.method, req.path);
+  logger.info(`${req.method}: ${req.path} - ${JSON.stringify(req.body)}`);
   next();
 };
+
+export const errorMiddleware: (err: ErrorRequestHandler, req: Request, resp: Response, next: NextFunction) => void = (
```

Comment:
File name is incorrect. Logger should be moved in loggerService (for example) and middlewares was be into separate files